draft-lenders-dns-over-coap.md

```diff
@@ -119,17 +119,13 @@ The terms "CoAP payload" and "CoAP body" are used as defined in [RFC7959].
 Selection of a DoC Server
 =========================
 
-TBD:
-
-- URI or should we rather go straight to CRI?
-- Support for more than one URI by DoC server.
-- DoC server identity, key exchange, ...
-
-URI Alternatives
-----------------
-TBD:
+It is assumed in this document that the DoC server, as well as the DNS query resource at the DoC
```

Comment:
In this document, it is assumed that the DoC client knows the DoC server and the DNS query at the DoC server. Possible options could be manual configurations of a URI {{?RFC3986}} or CRI {{?I-D.ietf-core-href}}, or automatic configuration using a  CoRE resource directory {{?I-D.ietf-core-resource-directory}}.

Reply:
> […] and the DNS query at the DoC server.

I guess you meant "the DNS resource" here?

Reply:
yes

Reply:
> […] or automatic configuration using a CoRE resource directory {{?I-D.ietf-core-resource-directory}}.

This somewhat excludes all other automatic configurations (DHCPv6 or RAs e.g.). How about

> […] or automatic configuration, e.g. using a CoRE resource directory {{?I-D.ietf-core-resource-directory}}.